test/actions_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule GitPair.ActionsTest do
+  use ExUnit.Case, async: true
+
+  import Mox
+
+  alias GitPair.Actions
+  alias GitPair.SystemMock
+
+  setup :verify_on_exit!
+
+  test ".add calls git config add comma" do
```

Comment:
What this test means?

Reply:
**praise:** nice catch! :baseball: 

Will fix tonight

Reply:
Done in a6d0f21